### PR TITLE
HAWQ-766. Add feature test for partition using new feature test framework

### PR DESCRIPTION
**Pull request by huor**

### CODE REVIEW

**Comment by yaoj2** [comment]
＋1

File: src/test/feature/partition/test-partition.cpp [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+class TestPartition: public ::testing::Test
+{
+	public:
+		TestPartition() {}
+		~TestPartition() {}
+};
+
+TEST_F(TestPartition, TestPartitionNegativeAndBasics)
+{
+	SQLUtility util;
```
Comment (xunzhang) [inline src/test/feature/partition/test-partition.cpp]: based on https://github.com/apache/incubator-hawq/pull/683, please use `hawq::test:: SQLUtility` here.

File: src/test/feature/partition/test-partition.cpp [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#include "gtest/gtest.h"
+
+#include "lib/sql-util.h"
```
Comment (xunzhang) [inline src/test/feature/partition/test-partition.cpp]: based on https://github.com/apache/incubator-hawq/pull/683, please write `lib/sql_util.h` here.

**Comment by huor** [comment]
Close this pull request. Move the code review to https://github.com/apache/incubator-hawq/pull/686 due to manual merge of latest test framework refactor.

**Pull request closed without merging**